tests/test_utils.py

```diff
@@ -0,0 +1,206 @@
+from lxml import objectify
+import pytest
+from django.forms import ValidationError
+
+from controlled_vocabularies import admin, views, vocabulary_handler
```

Comment:
Perhaps importing the `VocabularyHandler` from `vocabular_handler` would be beneficial. It appears that is the only thing we are using from that module.